src/main/java/com/github/jasminb/jsonapi/ResourceConverter.java

```diff
@@ -773,6 +774,13 @@ private ObjectNode getDataNode(Object object, Map<String, ObjectNode> includedCo
 		}
 		dataNode.set(ATTRIBUTES, attributesNode);
 
+		final PropertyNamingStrategy namingStrategy;
```

Comment:
Naming strategy should be a class variable and created in constructor that handles the 'ObjectMapper'.